Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Entities.mustache

```diff
@@ -233,7 +233,7 @@ namespace {{AggregateNamespace}}
         //                     One-to-one relationships
         // -------------------------------------------------------------
             {{#OneToOnes}}
-        [ValidateObject]
+        {{#IsRequired}}[Required]{{/IsRequired}}[ValidateObject]
```

Comment:
Remove this change if you don't want to enforce required on non-extension embedded objects.